### PR TITLE
Finished Spanish Translations😳

### PR DESCRIPTION
**Pull request by Vexot**
arriba Starboard 😳

### CODE REVIEW

**Comment by Vexot** [comment]
Oops, meant to add this to #15

**Comment by TheNoob27** [comment]
I'll be closing the other PR since you've made all the progress here

File: index.js [hunk 1]
```diff
@@ -54,6 +54,7 @@
  * @typedef {
     "en-GB" |
     "en-US" |
+    "es"    |
```
Comment (TheNoob27) [inline index.js:57]:
```suggestion
    "es-ES"    |
```

File: index.js [hunk 2]
```diff
@@ -86,6 +87,7 @@ if (typeof String.prototype.stripIndents !== "function") Object.defineProperties
  * @type {[
     "en-GB",
     "en-US",
+    "es",
```
Comment (TheNoob27) [inline index.js:90]:
```suggestion
    "es-ES",
```

File: languages/en-GB.js [hunk 3]
```diff
@@ -1142,6 +1142,7 @@ module.exports = {
   LANGUAGES: {
     EN_GB: "English (GB)",
     EN_US: "English (US)",
+    ES: "Spanish",
```
Comment (TheNoob27) [inline languages/en-GB.js:1145]:
```suggestion
    ES_ES: "Spanish",
```

File: languages/es.js [hunk 4]
```diff
@@ -0,0 +1,1138 @@
...
+      FAIL: "Algo salió mal al enviarte ese mensaje.",
+      NO_MESSAGE_ID: "Por favor provee un ID de mensaje."
+    }
+  }, // might alphabetically order the commands one day -- not needed
+
+    // languages
+    LANGUAGES: {
+      EN_GB: "Inglés (Reino Unido)",
+      EN_US: "Inglés (Estados Unidos)",
+      ES: "Español",
```
Comment (TheNoob27) [inline languages/es.js:1133]:
```suggestion
      ES_ES: "Español",
```

File: languages/es.js [hunk 5]
```diff
@@ -0,0 +1,1138 @@
+const LOCALE = "es"
```
Comment (TheNoob27) [inline languages/es.js:1]:
```suggestion
const LOCALE = "es-ES"
```

**Review by TheNoob27 — CHANGES_REQUESTED** [review]
Thank you for your contribution! It'd be preferred if the locale were to be `es-ES` instead of `es`, to match other languages. Please change the file name too

**Comment by Vexot** [comment]
The requested changes were made

File: languages/es-ES.js [hunk 6]
```diff
@@ -0,0 +1,1138 @@
...
+      SAVED: "Mensajes Enviados A MDs",
+      QUICK_ACTIONS: "Acciones Rápidas Efectuadas",
+      CPU: "CPU",
+      DISK_USAGE: "Uso del Disco",
+      MEMORY_USAGE: "Uso de Memoria",
+      PROCESS_UPTIME: "Tiempo de Funcionamiento del Proceso",
+      PLATFORM: "Platforma",
+      LIBRARY: "Libreria"
+    },
+    HOW: {
```
Comment (TheNoob27) [inline languages/es-ES.js:1015]:
```suggestion
    SHOW: {
```

**Review by TheNoob27 — CHANGES_REQUESTED** [review]

**Comment by TheNoob27** [comment]
for each command, you can actually add a NAME key to give the command a spanish alias if you'd like 😁

for example:
```js
    HELP: {
      NAME: "ayuda",
      DESCRIPTION: "...",
      USAGE: "ayuda ([comando])"
```
so `star help` and `star ayuda` would then work! you can feel free to do so

**Review by TheNoob27 — APPROVED** [review]

**Pull request merged**